views/index.html

```diff
@@ -18,28 +18,48 @@
 
 <body>
   <header class="toolbar">
-    <img class="toolbar-image" src="../assets/images/toolbar_mobile_img.png" alt="toolbar image">
+    <img class="toolbar-image" src="../assets/images/IntroSection/toolbar_mobile_img.png" alt="toolbar image">
     <div class="icon-wrapper"><i class="fas fa-bars"></i></div>
+    <nav class="nav-bar">
+      <div class="nav-bar-mobile-top"></div>
+      <ul class="nav-links">
+        <li class="nav-link"><a href="#Hello">Hello</a></li>
+        <li class="nav-link"><a href="#Portfolio">Portfolio</a></li>
+        <li class="nav-link"><a href="#About">About</a></li>
+        <li class="nav-link"><a href="#Contact">Contact</a></li>
+      </ul>
+      <div class="nav-bar-mobile-bottom"></div>
+    </nav>
   </header>
 
   <main>
     <!-- ************************************************************************************************** -->
     <!-- *****                                        INTRO SECTION                                   ***** -->
     <!-- ************************************************************************************************** -->
 
-    <section class="intro-section margin-x">
-      <h1 class="display-1">Leonardo Albornoz</h1>
-      <p class="par">Hello! I am a software developer! I can help you build a product, feature or website. Taka a look
-        of my works. If you like what you see and have a project you need coded, don’t hesitate and contact me.</p>
-      <button class="btn" type="button"><span>Start collaboration</span></button>
+    <section id="Hello" class="intro-section margin-top">
```

Comment:
- [ ] [OPTIONAL] You have done an excellent job with your desktop version design :muscle: but there is a spacing/layout issue with the mobile version because it's looking too small considering the Figma design. I will mark this as [OPTIONAL] because it's a minor change but I strongly suggest fixing this issue :wink: :raised_hands:

  Your design:
  ![image](https://user-images.githubusercontent.com/81522435/144463909-bad45695-9658-4932-a932-17ed4c60ff2f.png)

  Figma design:
  ![image](https://user-images.githubusercontent.com/81522435/144464000-7a795ec8-802a-4118-bb91-da3598fc6ad1.png)